src/test/java/teammates/test/pageobjects/FeedbackSubmitPage.java

```diff
@@ -196,4 +198,22 @@ public void waitForOtherOptionTextToBeClickable(int qnNumber, int responseNumber
                 By.cssSelector("input[id$='OptionText-" + qnNumber + "-" + responseNumber + "']"));
         waitForElementToBeClickable(element);
     }
+    
+    // ------------- For InstructorEditStudentFeedbackPage -------------
+    
+    public boolean clickModerationHintButton() {
+        click(By.id("moderationHintButton"));
+        // Check if links toggle properly.
+        WebElement moderationHint = browser.driver.findElement(By.id("moderationHint"));
+        return moderationHint.isDisplayed();
```

Comment:
@JosephineKwa Changed Currently version seems more clear. Should I create another issue on `InstructorFeedbackResultsPageUiTest` to refactor the test case as well :)